server/app/controllers/core.controller.js

```diff
@@ -9,6 +10,92 @@ const {
 } = require("../models/core/destination.model")
 
 const F5_HOST_IP = '10.206.40.100';
+const DESTINATION_MAPPING_SCRIPT_PATH = '../scripts/get_all_destination_names.py';
+const DESTINATION_MAPPING_FILE = 'controller_info.json';
+
+const parseDestinationMappingsData = (req, res, destinationScriptResponse) => {
+    const { tenants, clouds } = destinationScriptResponse;
+
+    if(!Array.isArray(tenants) || !Array.isArray(clouds)) {
+        return res.status(500).json({ error: 'Something went wrong while parsing destination mapping data.'});
+    }
+
+    const destinationMappings = {
+        'tenants': []
```

Comment:
Can we divide this into multiple function, bit clear for debugging and things like that